src/lib/getCollectionBalanceOf.tsx

```diff
@@ -0,0 +1,40 @@
+import { COLLECTIONS } from "@/lib/consts"
+import { getPublicClient } from "@/lib/clients"
+import zora1155Abi from "@/lib/abi/zora-drop.json"
```

Comment:
please use ABI exports directly from `zora-sdk` package

Reply:
once you use the `npm` package export, please remove this ABI from our codebase 🚮